### PR TITLE
Fixes #239 Correct stage to stage_file in cmd_run.py

### LINKED ISSUE #239: cli: shows stack traceback when .err file is not find
When I do `popper run` with an invalid `validate.sh` file in a pipeline, the build fails and the following stack traceback is shown:

```
Executing mypipeline
setup.sh .
run.sh .
validate.sh .
Stage validate failed.
Logs for validate:.
Traceback (most recent call last):
  File "/home/ankan/Projects/popper-test/bin/popper", line 11, in <module>
    sys.exit(cli())
  File "/home/ankan/Projects/popper-test/lib/python3.6/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/home/ankan/Projects/popper-test/lib/python3.6/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/home/ankan/Projects/popper-test/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/ankan/Projects/popper-test/lib/python3.6/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/ankan/Projects/popper-test/lib/python3.6/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/home/ankan/Projects/popper-test/lib/python3.6/site-packages/click/decorators.py", line 64, in new_func
    return ctx.invoke(f, obj, *args[1:], **kwargs)
  File "/home/ankan/Projects/popper-test/lib/python3.6/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/home/ankan/Projects/popper-test/lib/python3.6/site-packages/popper/commands/cmd_run.py", line 51, in cli
    time_out, skip)
  File "/home/ankan/Projects/popper-test/lib/python3.6/site-packages/popper/commands/cmd_run.py", line 97, in run_pipeline
    with open('popper_logs/{}{}'.format(stage, t), 'r') as f:
FileNotFoundError: [Errno 2] No such file or directory: 'popper_logs/validate.err'
```

Is this the desired behavior?  Wouldn't we want to show a custom error message instead of showing the whole stack traceback?

### PR DESCRIPTION
**Pull request by ankan17**
When the popper run command was executed on a pipeline with an invalid stage, python stack traceback was shown for the error where because the code looked for something like validate.err file whereas the file generated is validate.sh.err. This change fixes the error.

### CODE REVIEW

File: cli/popper/commands/cmd_run.py [hunk 1]
```diff
@@ -91,11 +91,11 @@ def run_pipeline(project_root, pipeline, timeout, skip):
         ecode = execute(stage_file, timeout)
 
         if ecode != 0:
-            pu.info("Stage {} failed.".format(stage))
+            pu.info("Stage {} failed.".format(stage_file))
```
Comment (ivotron) [inline cli/popper/commands/cmd_run.py]: this is fine with `stage`

File: cli/popper/commands/cmd_run.py [hunk 2]
```diff
@@ -91,11 +91,11 @@ def run_pipeline(project_root, pipeline, timeout, skip):
...
             STATUS = "FAIL"
-            pu.info("Logs for {}:.".format(stage))
+            pu.info("Logs for {}:.".format(stage_file))
```
Comment (ivotron) [inline cli/popper/commands/cmd_run.py]: as well as this

**Comment by ankan17** [comment]
Changes have been made

**Comment by ivotron** [comment]
thanks!

**Pull request merged**